src/util/execute-lifecycle-script.js

```diff
@@ -195,7 +195,12 @@ export async function makeEnv(
       pathParts.unshift(path.join(config.workspaceRootFolder, binFolder));
     }
     pathParts.unshift(path.join(config.linkFolder, binFolder));
-    pathParts.unshift(path.join(cwd, binFolder));
+
```

Comment:
We could narrow this by checking to see if os is windows first _and/or_ stat to see if it's a symlink.